code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -366,7 +366,7 @@ var/global/list/parasites = list() //all currently existing/living guardians
 	var/list/possible_guardians = list("Chaos", "Standard", "Ranged", "Support", "Explosive", "Lightning", "Protector", "Charger", "Assassin")
 	var/random = TRUE
 	var/allowmultiple = 0
-	var/allowling = 0
+	var/allowling = 1
```

Comment:
Oh nevermind I got confused by Coiax's comment, I see this is still just all stands.

>5th PR trying to do this

wake me up